frontend/src/components/activity/CopyActivityInfoDialog.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <dialog-form
+    v-model="showDialog"
+    icon="mdi-content-copy"
+    :title="$tc('components.activity.copyActivityInfoDialog.title')"
+    :cancel-action="cancel"
+    :cancel-label="$tc('global.button.close')"
+  >
+    <template #activator="scope">
+      <slot name="activator" v-bind="scope" />
+    </template>
+
+    <p>
+      {{ $tc('components.activity.copyActivityInfoDialog.description') }}
+    </p>
+    <p v-if="clipboardReadState === 'prompt'">
+      <center>
+        <v-btn color="success" @click="requestClipboardAccess">
+          {{ $tc('components.activity.copyActivityInfoDialog.allow') }}
+        </v-btn>
+      </center>
+    </p>
+    <p v-if="clipboardReadState === 'granted'">
+      {{ $tc('components.activity.copyActivityInfoDialog.granted') }}
+    </p>
+    <p v-if="clipboardReadState === 'denied'">
```

Comment:
minor:
Could we keep the button to request access also in the denied case?
On some browsers (e.g. mobile) the button to grant the permission is not that easy to find.